pkg/network/link/address_test.go

```diff
@@ -101,4 +102,18 @@ var _ = Describe("Common Methods", func() {
 			Expect(mac).To(Equal(&expectedMac))
 		})
 	})
+	Context("GetFakeBridgeIP function", func() {
+		It("Should return empty string when interface name is not in the interface list", func() {
+			ip := GetFakeBridgeIP([]v1.Interface{v1.Interface{Name: "aaaa"}}, &v1.Interface{Name: "abcd"})
+			Expect(ip).To(Equal(""))
+		})
+		It("Should return the correct ip when the interface is the first in the list", func() {
+			ip := GetFakeBridgeIP([]v1.Interface{v1.Interface{Name: "abcd"}}, &v1.Interface{Name: "abcd"})
+			Expect(ip).To(Equal(fmt.Sprintf(bridgeFakeIP, 0)))
```

Comment:
do you mind specifying the IP manually ? e.g. without relying on the bridge IP template ? Do you think that's a good idea ?

Reply:
If I will specify it manually, changing `bridgeFakeIP` will break the test.
I see the `bridgeFakeIP` as part of the method contract. The method takes `bridgeFakeIP` and inserts the correct number to the pattern, depending on the order of the interfaces.

Reply:
> If I will specify it manually, changing `bridgeFakeIP` will break the test.

Exactly. That's the objective.

> I see the `bridgeFakeIP` as part of the method contract. The method takes `bridgeFakeIP` and inserts the correct number to the pattern, depending on the order of the interfaces.

I guess I see it differently; if it were part of the method contract, it should be *in* the method.

Anyway, this is peanuts, it's not worth our time to bicker over it.